File: generated-sources/php-slim/mojang-authentication/lib/com/github/asyncmc/mojang/authentication/php/slim/model/AuthenticationRequest.php
<?php
/**
 * AuthenticationRequest
 */
namespace com.github.asyncmc.mojang.authentication.php.slim.server\com.github.asyncmc.mojang.authentication.php.slim.model;

/**
 * AuthenticationRequest
 */
class AuthenticationRequest
{
    
    /** @var string $username The Mojang account e-mail or username. Never store it.*/
    private $username;
    
    /** @var string $password The Mojang account password, never store it.*/
    private $password;
}

File: generated-sources/dart-jaguar/mojang-authentication/lib/model/authentication_request.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:openapi/model/username_password.dart';
part 'authentication_request.jser.dart';

class AuthenticationRequest {
   /* The Mojang account e-mail or username. Never store it. */
  @Alias('username')
  final String username;
   /* The Mojang account password, never store it. */
  @Alias('password')
  final String password;
  

  AuthenticationRequest(
    

{
    
     this.username = null,  
     this.password = null 
    }
  );

  @override
  String toString() {
    return 'AuthenticationRequest[username=$username, password=$password, ]';
  }
}

@GenSerializer()
class AuthenticationRequestSerializer extends Serializer<AuthenticationRequest> with _$AuthenticationRequestSerializer {

}
